components/RenderField/index.js

```diff
@@ -2,13 +2,10 @@ import { useTranslation } from 'react-i18next'
 
 export default function RenderField({ object, field, isHtml, maxSize }) {
   const { i18n } = useTranslation()
-  let content
+  const defaultLanguage = 'en'
```

Comment:
Should not have hardcoded default language to 'en'. This config should come from i18n definitions.